### PR TITLE
[ICDS Dashboard] back data filling for SAG

### PR DESCRIPTION
**Pull request by Rohit25negi**
This is the back data filling for out of school ag girls(A new feature).
This will have to be done for April-Dec.

### CODE REVIEW

File: custom/icds_reports/utils/data_filling.sql [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                                                   ->  Index Scan using icds_dashboard_adolescent_girls_registration_pkey_218364 on icds_dashboard_adolescent_girls_registration_218364 adolescent_girls_table  (cost=0.41..2.63 rows=1 width=73)
+                                                         Index Cond: (month = '2019-04-01'::date)
+(23 rows)
+*/
+
+UPDATE "agg_awc_2019-04-01_5" agg_awc
+    set cases_person_adolescent_girls_11_14_out_of_school = ut.girls_out_of_schoool,
+    cases_person_adolescent_girls_11_14_all_v2 = ut.cases_person_adolescent_girls_11_14_all_v2
+ FROM
+ (select * from tmp_sag_table) ut
```
Comment (calellowitz) [inline custom/icds_reports/utils/data_filling.sql]: this should be able to be rewritted as just `FROM tmp_sag_table ut`

File: custom/icds_reports/utils/data_filling.sql [hunk 2]
```diff
@@ -0,0 +1,119 @@
+CREATE UNLOGGED TABLE tmp_sag_table as (
+        select
+            ucr.awc_id,
+            ucr.supervisor_id,
+            SUM(CASE WHEN ( (out_of_school or re_out_of_school) AND
+                        (not admitted_in_school )) THEN 1 ELSE 0 END ) as girls_out_of_schoool,
+            SUM(CASE WHEN '2008-04-30' > dob AND '2005-04-01' <= dob AND sex = 'F' THEN 1 ELSE 0 END) as cases_person_adolescent_girls_11_14_all_v2
```
Comment (calellowitz) [inline custom/icds_reports/utils/data_filling.sql]: do we not already have this in agg_awc?

**Review by calellowitz — APPROVED** [review]
The first query looks nasty but i guess fine to run. left one other comment but nothing blocking

**Comment by Rohit25negi** [comment]
closing as this is done

**Pull request closed without merging**